js/activity.js

```diff
@@ -338,6 +338,43 @@ function Activity() {
         toolbarHeight = document.getElementById("toolbars").offsetHeight;
     };
 
+    // Checks which browser is MB running from
+
+    this.doBrowserCheck = function() {
```

Comment:
Maybe this function and the detectIE function should both live in utils/platformstyle.js ???

Reply:
I've moved this function inside `utils` and the corresponding check inside `activity` has been displaced to `init()` function.

Reply:
I've also tried moving detectIE to utils and doing the necessary changes in `sw.js` and readme. 
Please check.